homeassistant/components/switch/__init__.py

```diff
@@ -30,6 +30,11 @@
 
 ATTR_TODAY_MWH = "today_mwh"
 ATTR_CURRENT_POWER_MWH = "current_power_mwh"
+# ATTR_CURRENT_POWER_W = "Current power (W)"
+ATTR_CURRENT_POWER_W = "current_power_w"
```

Comment:
Please do not add constants to the main switch component that will only be used inside your platform